riocli/v2client/client.py

```diff
@@ -119,7 +119,12 @@ def create_project(self, spec: dict) -> Munch:
         Create a new project
         """
         url = "{}/v2/projects/".format(self._host)
-        headers = self._get_auth_header(with_project=False)
+        headers = self._get_auth_header(with_project=False, with_org=True)
```

Comment:
Looks like for update and delete also with_org is not set. Can we set that as well?

Reply:
I've updated the client. Please take a look.